specs/receive_funds/03_technical_details.rst

```diff
@@ -0,0 +1,16 @@
+=========================
+Technical Details
+=========================
+
+=====  =============  =========  ==========
+ 68        title      tschubotz   created
+=====  =============  =========  ==========
+`68`_  Receive funds  tschubotz  2019-01-17
+=====  =============  =========  ==========
+
+.. _68: https://github.com/gnosis/safe/issues/68
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
```

Comment:
We have to put here the specification for the QR code sharing format.

Should it just address or using a [EIP-831](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-831.md) format?

What text should be contained in the QR-code? Checksummed address?

Reply:
I agree but shall we create a different spec for this? If the same restrictions will be applied across the app then we can just reference the new spec for displaying addresses.